x-pack/test/functional_synthetics/services/uptime/synthetics_package.ts

```diff
@@ -50,6 +50,7 @@ export function SyntheticsPackageProvider({ getService }: FtrProviderContext) {
         apiRequest = retry.try(() => {
           return supertest
             .get(INGEST_API_EPM_PACKAGES)
+            .query({ experimental: true })
```

Comment:
@jen-huang Does this need to be ported forward to `main` as well since this is an 8.1 PR?

Reply:
As Jen noted above as well in https://github.com/elastic/kibana/pull/126193#discussion_r812410530, it looks like it's already there on `main` https://github.com/elastic/kibana/blob/3195a544067b35f36ff41ca5c538fa2c3bf2de21/x-pack/test/functional_synthetics/services/uptime/synthetics_package.ts#L53